telluric/features.py

```diff
@@ -9,6 +14,48 @@
 from telluric.plotting import NotebookPlottingMixin
 
 
+def transform_attributes(attributes, schema):
+    """Transform attributes types according to a schema.
+
+    Parameters
+    ----------
+    attributes : dict
+        Attributes to transform.
+    schema : dict
+        Fiona schema containing the types.
+
+    """
+    new_attributes = attributes.copy()
+    for attr_value, (attr_name, attr_type) in zip(new_attributes.values(), schema["properties"].items()):
+        if attr_type == "date":
+            new_attributes[attr_name] = parse_date(attr_value).date()
+        elif attr_type == "datetime":
+            new_attributes[attr_name] = parse_date(attr_value)
+
+    return new_attributes
+
+
+def serialize_attributes(attributes):
+    """Serialize attributes.
+
+    Parameters
+    ----------
+    attributes : dict
+        Attributes to serialize.
+
+    """
+    encoder = JSONEncoder()
+    new_attributes = attributes.copy()
+    for attr_name, attr_value in new_attributes.items():
+        try:
+            encoder.encode(attr_value)
```

Comment:
This looks like a expensive way to check if the attributes needs a special treatment

Reply:
My plan B was to do

```
if not isinstance(attr_value, (dict, list, tuple, str, int, float, bool, type(None))):
    ...
```

(see https://docs.python.org/3.4/library/json.html#json.JSONEncoder)

Do you think it's better?

Reply:
it does not look nice but programatically it is better, wdyt?

Reply:
I agree, will do the change.